src/logs/logs_sender.py

```diff
@@ -23,10 +23,11 @@
 
 LOGS_API_PATH = "/api/v2/logs/ingest"
 
+DYNATRACE_LOG_INGEST_CONTENT_MARK_TRIMMED = "[TRUNCATED]"
 DYNATRACE_LOG_INGEST_CONTENT_MAX_LENGTH = 8192
 DYNATRACE_LOG_INGEST_ATTRIBUTE_MAX_LENGTH = 250
-DYNATRACE_LOG_INGEST_REQUEST_MAX_SIZE = 1048576 # 1MB
-DYNATRACE_LOG_INGEST_MAX_RECORD_AGE = 86340 # 1 day
+DYNATRACE_LOG_INGEST_REQUEST_MAX_SIZE = 1048576  # 1MB
+DYNATRACE_LOG_INGEST_MAX_RECORD_AGE = 86340  # 1 day
```

Comment:
I was wondering what is this change about, but I assume it was autoformatted. I found this in Python docs:
_An inline comment is a comment on the same line as a statement. Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space._
https://www.python.org/dev/peps/pep-0008/#inline-comments

Reply:
Yes, that's the reason I left this change. PEP 8: E261 at least two spaces before inline comment. Can I leave it?

Reply:
Sure. It's better to follow standards 🙂